### ISSUE TITLE
sandbox can be undefined in EditorPreview's render()

### ISSUE THREAD

**Issue by AlixWang** [codesandbox/codesandbox-client#2770]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System |
  | ------- | --------- | ---------------- |
  | chrome | 77.0.3865 | Windows 10           |

  **Route:**
  https://codesandbox.io/s/bold-waterfall-cplk4

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'template' of undefined
  TypeError: Cannot read property 'template' of undefined
    at n.value (https://codesandbox.io/static/js/app.7b7fc602.js:438:433874)
    at https://codesandbox.io/static/js/default~app~embed.959d3548.chunk.js:276:354493
    at f.trackScope (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1468208)
    at n.<anonymous> (https://codesandbox.io/static/js/default~app~embed.959d3548.chunk.js:276:354462)
    at vs (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2063034)
    at ms (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2062827)
    at Ka (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2097223)
    at Fa (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2081589)
    at Na (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2078609)
    at Ra (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2077169)
    at https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2039070
    at https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2113916
    at co (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2038799)
    at po (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2039016)
    at ho (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2038951)
    at Ia (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2077596)
    at https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2028438
    at HTMLDocument.r (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:122867)
  
    in n
    in r
    in div
    in t
    in div
    in t
    in div
    in div
    in t
    in styled.div
    in div
    in t
    in elements__Container
    in t
    in n
    in r
    in n
    in r
    in t
    in t
    in div
    in t
    in elements__Content
    in n
    in t
    in withRouter(n)
    in div
    in t
    in elements__Container
    in Unknown
    in t
    in withRouter()
    in DragDropContext(withRouter())
    in t
    in t
    in l
    in t
  ```

  </details>

**Comment by jepser** [comment]
I don't see anything here @AlixWang if you could bring more info it will be useful.

**Comment by CompuIves** [comment]
I saw this too, but it seems like an impossible state because that component shouldn't even render if currentSandbox is undefined.

**Comment by lbogdan** [comment]
Closing this, it should have been fixed in #3583 :

https://github.com/codesandbox/codesandbox-client/blob/66714e552553a38475a9386e34b281a201dbddb1/packages/app/src/app/pages/Sandbox/Editor/Content/index.tsx#L87
